docs/url-formatter.asciidoc

```diff
@@ -0,0 +1,26 @@
+The `Url` field formatter can take on the following types:
+
+* The *Link* type turn the contents of the field into an URL. 
+* The *Image* type can be used to // what?
```

Comment:
It's pretty unclear from the K4UI what the Image formatter does, and I could use a quick explainer.